module/VuFind/src/VuFind/Regex/Regex.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Allow to match regex from config file
+ *
+ * PHP version 8
+ *
+ * @category VuFind
+ * @package  Regex
+ * @author   MSUL Public Catalog Team <LIB.DL.pubcat@msu.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/vufind/ Main page
+ */
+
+namespace VuFind\Regex;
+
+use Exception;
+
+use function array_key_exists;
+
+/**
+ * Class to hold data for the Get This button
+ *
+ * @category VuFind
+ * @package  Regex
+ * @author   MSUL Public Catalog Team <LIB.DL.pubcat@msu.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/vufind/ Main page
+ */
+class Regex
+{
+    /**
+     * Initializes the loader
+     *
+     * @param array $config Config pulled from the file
```

Comment:
Maybe this would be more descriptive:

```suggestion
     * @param array $config Regular expression configuration
```